lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -228,6 +252,21 @@ click_dropdown_field:
   - selector:
       xpath: //*[(.='<dropdown_field>')]/following-sibling::*//button[contains(@class, 'dropdown__toggle')]
     op: click
+click_first_item_in_resource_list:
+  elements:
+  - selector:
+      xpath: //td[1]//span[@class='co-resource-item']/a
```

Comment:
I tried this rule, it focused on the first column for all lines, not exact the first line, you should use:
//tr[1]/td[1]//span[@class='co-resource-item']/a

Reply:
updated